README.md

```diff
@@ -24,6 +24,7 @@ Commands:
   validation
   chain
   key
+  conversion
```

Comment:
is it needed?

Reply:
No, it doesn't need after `conversion` merged into under `key` subcommand. I will remove it.

Reply:
I removed it in 6acc9ac.